### PR TITLE
REL-327: prepare release notes

### PR DESCRIPTION
**Pull request by oliverhowell**
Preview link workaround: https://deploy-preview-1281--hardcore-allen-f5257d.netlify.app/hazelcast/5.5/release-notes/5-5-1

### CODE REVIEW

File: docs/modules/release-notes/pages/5-5-1.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+{description}
+
+== Known Issues
+* There is currently a known issue that has potential for causing a memory leak in Vector collections in some scenarios. For more info, including a workaround, see xref:data-structures:vector-collections.adoc#known-issue[Vector Collection Known Issue].
+
+== Fixes
+* Fix partition table snapshot selection 
+* Fix some invocations not de-registered due to invocation method
+* Fix ClassCastException when Offloadable EntryProcessor used with Interceptor
```
Comment (JackPGreen) [inline docs/modules/release-notes/pages/5-5-1.adoc:12]:
```suggestion
* Fix `ClassCastException` when Offloadable EntryProcessor used with Interceptor
```

File: docs/modules/release-notes/pages/5-5-1.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+* Fix HD memory exception with TPC
+* Disable TPC if ALL_MEMBERS routing is not configured
```
Comment (JackPGreen) [inline docs/modules/release-notes/pages/5-5-1.adoc:14]:
```suggestion
* Disable Thread-Per-Core (TPC) if `ALL_MEMBERS` routing is not configured
```

consistency with https://docs.hazelcast.com/hazelcast/latest/cluster-performance/thread-per-core-tpc

File: docs/modules/release-notes/pages/5-5-1.adoc [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+{description}
+
+== Known Issues
+* There is currently a known issue that has potential for causing a memory leak in Vector collections in some scenarios. For more info, including a workaround, see xref:data-structures:vector-collections.adoc#known-issue[Vector Collection Known Issue].
+
+== Fixes
+* Fix partition table snapshot selection 
+* Fix some invocations not de-registered due to invocation method
+* Fix ClassCastException when Offloadable EntryProcessor used with Interceptor
+* Fix HD memory exception with TPC
```
Comment (JackPGreen) [inline docs/modules/release-notes/pages/5-5-1.adoc:13]:
```suggestion
* Fix HD memory exception with Thread-Per-Core (TPC)
```
Consistency with https://docs.hazelcast.com/hazelcast/latest/cluster-performance/thread-per-core-tpc

File: docs/modules/release-notes/pages/5-5-1.adoc (hunk 2 above)
Comment (JackPGreen) [inline docs/modules/release-notes/pages/5-5-1.adoc:14]:
```suggestion
* Disable TPC if `ALL_MEMBERS` routing is not configured
```

File: docs/modules/release-notes/pages/5-5-1.adoc [hunk 4]
```diff
@@ -0,0 +1,14 @@
+= 5.5.1 Release Notes
+:description: These release notes list any new features, enhancements, fixes, and breaking changes implemented between version 5.5.0 and version 5.5.1 of Hazelcast Platform.
+
+{description}
+
+== Known Issues
+* There is currently a known issue that has potential for causing a memory leak in Vector collections in some scenarios. For more info, including a workaround, see xref:data-structures:vector-collections.adoc#known-issue[Vector Collection Known Issue].
+
+== Fixes
+* Fix partition table snapshot selection (SUP-538)
```
Comment (g-donev) [inline docs/modules/release-notes/pages/5-5-1.adoc:10]: My suggestion: 
Partitions are not distributed correctly after all Members in a partition group are restarted and persistence is enabled

**Pull request merged**